samples/gke-monitoring/README.md

```diff
@@ -39,7 +39,7 @@ volume mount so that the sidecar can directly read the Prometheus server data.
 Although there are several ways to deploy the sidecar to a Prometheus pod,
 this sample directly supports only
 [Prometheus custom resources](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#prometheus)
-such as those deployed by the SAS Viya monitoring components in
+such as those deployed by the SAS Viya platform monitoring components in
```

Comment:
This is a reference to the SAS Viya Monitoring project and not to SAS Viya (itself); therefore addition of "platform" seems inappropriate.